Sources/Command/Command.swift

```diff
@@ -124,12 +124,18 @@ public enum CommandEvent: Sendable {
 
     public var utf8String: String {
         switch self {
-        case let .standardOutput(bytes):
-            String(decoding: bytes, as: Unicode.UTF8.self)
-        case let .standardError(bytes):
+        case let .standardOutput(bytes),
+             let .standardError(bytes):
             String(decoding: bytes, as: Unicode.UTF8.self)
         }
     }
+
+    public var isError: Bool {
+        switch self {
+        case .standardError: true
+        default: false
```

Comment:
Any reason to not use `case .standardOutput` instead of `default`?
So in case we add new cases we are sure we double check this logic

Reply:
Good point